packages/nns/src/governance.canister.ts

```diff
@@ -357,6 +381,28 @@ export class GovernanceCanister {
     });
   };
 
+  /**
+   * Sets node provider account
+   *
+   * @param {accountIdentifier}
```

Comment:
the `accountIdentifier` it isn't an object

```
* @param accountIdentifier An account that should be set as node provider
```

or

```
* @param {string} accountIdentifier An acco...
```

Reply:
ok

Reply:
Is there a link for me to know exactly how to document those? I honestly don't know how it's done. I feel I do one thing in one part and another in another one.

Reply:
https://tsdoc.org/ is a good source but generally I just check the other comments

Reply:
thanks!

Reply:
done